src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+/* region card */
```

Comment:
It's better to delete it before pushing. + Anyway you have separate files for each block